pulsar-client-reactive-jackson/src/test/java/org/apache/pulsar/reactive/client/jackson/PulsarReactiveClientModuleTests.java

```diff
@@ -493,6 +508,13 @@ void shouldSerDeserEmptyDeadLetterPolicy() throws Exception {
 		assertThat(json).isEqualTo("{\"maxRedeliverCount\":0}");
 	}
 
+	@Test
+	void shouldSerDeserTestRedeliveryBackoff() throws Exception {
+		TestRedeliveryBackoff backoff = MAPPER.readValue("{}", TestRedeliveryBackoff.class);
+		String json = MAPPER.writeValueAsString(backoff);
+		assertThat(json).isEqualTo("{\"className\":\"org.apache.pulsar.reactive.client.jackson.PulsarReactiveClientModuleTests$TestRedeliveryBackoff\"}");
```

Comment:
For consistency w/ the other tests do you mind using the class get name here as well?
```java
String expected = ("{'className':'" + TestRedeliveryBackoff.class.getName() + "'}").replaceAll("'", "\"");
assertThat(json).isEqualTo(expected);
```

Reply:
Yes, fixed.